src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/PathSeqKmerSpark.java

```diff
@@ -13,7 +13,10 @@
 import org.broadinstitute.hellbender.cmdline.programgroups.SparkProgramGroup;
 import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
 import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
-import org.broadinstitute.hellbender.tools.spark.sv.*;
```

Comment:
Nope. It was caused by the "repackaging" of classes that was taken out in this PR.